metaflow/plugins/conda/batch_bootstrap.py

```diff
@@ -46,6 +49,13 @@ def install_conda_environment(env_id, packages):
         'conda create --yes --no-default-packages -p {0} --no-deps {1} >/dev/null 2>&1'.format(os.path.join(os.getcwd(), env_id), ' '.join(packages)),
         'cd {0}'.format(os.getcwd())
     ]
+    if ESCAPE_HATCH_PY is not None:
+        cwd = os.getcwd()
+        generate_trampolines(ESCAPE_HATCH_PY, cwd)
```

Comment:
nit - could this function simply be  `generate_trampolines(cwd)` given that `ESCAPE_HATCH_PY` is already defined in `env_escape`

Reply:
i'll check that works but if so, sure. No issues for me there.